components/SnippetCard.js

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+
+export default function SnippetCard({ snippetData, name, description }) {
+  if (!snippetData) return <div>No snippets yet 😭</div>;
+  return (
+    <StyledCard>
+      <CardHeading>{name}</CardHeading>
+      <CardDescription>{description}</CardDescription>
+    </StyledCard>
+  );
+}
+
+const StyledCard = styled.div`
```

Comment:
you could change this to be a section:

```suggestion
const StyledCard = styled.section`
```